release.nix

```diff
@@ -8,15 +8,17 @@ let
   ethereum-test-suite = x: x.fetchFromGitHub {
     owner = "ethereum";
     repo = "tests";
-    rev = "7e361956bd68f5cac72fe41f29e5734ee94ae2de";
-    sha256 = "0l5qalgbscr77vjhyf7b542055wnp4pddpfslnypp5sqws5w940w";
+    rev = "da6d391922cb0e3c6bda24871c89d33bc815c1dc";
+    sha256 = "06h3hcsm09kp4hzq5sm9vqkmvx2nvgbh5i788qnqh5iiz9fpaa9k";
   };
 
-  hevmTestReport = x: x.runCommand "hevm-test-report" {} ''
+  hevmCompliance = x: x.runCommand "hevm-compliance" {} ''
     mkdir -p $out/nix-support
     export PATH=${x.pkgs.hevm}/bin:$PATH
-    ${x.pkgs.hevm}/bin/hevm vm-test-report \
-      --tests ${ethereum-test-suite x} > $out/index.html
+    ${x.pkgs.hevm}/bin/hevm compliance \
+      --tests ${ethereum-test-suite x} \
+      --skip modexp \
+      --html > $out/index.html
```

Comment:
I think we don't need to save the html anymore. This was used when we had hydra, but now isn't served anywhere.